src/Utils/MockData/BackendMock.js

```diff
@@ -10,7 +13,10 @@ class BackendMock {
 			.set(/^entitytype\/hierarchy$/i, BackendMock.getEntityTypeHierarchy)
 			.set(/^entitytype\/(-?\d+)\/attributes$/i, BackendMock.getEntityTypeAttributes)
 			.set(/^entity\/(-?\d+)\/children\/type\/(-?\d+)\/(((\w)+)\+)*(\w)*$/i, BackendMock.getChildEntitiesOfEntityType)
-			.set(/^entity\/(-?\d+)$/i, BackendMock.getEntity);
+			.set(/^entity\/(-?\d+)$/i, BackendMock.getEntity)
+            .set(/^entity\/(-?\d+)\/eventtypes$/i, BackendMock.getEventTypesOfEntity)
+            .set(/^eventtype\/(-?\d+)\/attributes$/i, BackendMock.getAttributesOfEventType)
+            .set(/^entity\/(-?\d+)\/eventtype\/(-?\d+)\/events/i, BackendMock.getEventsOfEventTypeAndEntity);
```

Comment:
Problem here is: The api offers offsets at the end of the url but we want to cover all events-api calls, even with set offsets with this regex.
As there shouldn't be any other route starting with `entity/id/eventtype/id/events` there shouldn't be a conflict.